api/environments/identities/tests/test_views.py

```diff
@@ -174,6 +177,26 @@ def test_can_search_for_identities(self):
         # and - only identity matching search appears
         assert res.json().get("count") == 1
 
+    def test_can_search_for_identities_with_exact_match(self):
+        # Given
+        Identity.objects.create(identifier="1", environment=self.environment)
+        Identity.objects.create(identifier="12", environment=self.environment)
+        Identity.objects.create(identifier="121", environment=self.environment)
+        base_url = reverse(
+            "api-v1:environments:environment-identities-list",
+            args=[self.environment.api_key],
+        )
+        url = "%s?%s" % (base_url, urllib.parse.urlencode({"q": '"1"'}))
+
+        # When
+        res = self.client.get(url)
+
+        # Then
+        assert res.status_code == status.HTTP_200_OK
+
+        # and - only identity matching search appears
+        assert res.json().get("count") == 1
```

Comment:
Let's test that it's the _correct_ result.

You'll want to change at least one of the identities above to a named variable and then do something like: 

```python
identity_to_return = Identity.objects.create(
    identifier="1", environment=self.environment
)
...

# and - only the expected identity matching search appears
response_json = res.json()
assert response_json.get("count") == 1
assert response_json["results"][0]["id"] == identity_to_return.id
```